src/components/base/containers/DrawerPanel.js

```diff
@@ -1,9 +1,41 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
 import Icon from '../icons/Icon';
+import { colors } from '../../theme';
+import styled from 'styled-components';
 import Collapse from 'react-smooth-collapse-customizable';
 
+const PanelWrapper = styled.div`
+  border-bottom: 1px solid ${colors.grayLight};
+`;
+
+const PanelHeader = styled.div`
+  cursor: pointer;
+  padding: 10px 15px;
+
+  &:hover {
+    background-color: ${colors.grayLighter};
+    color: ${colors.wtwViolet};
```

Comment:
surprised this is working. this should be ``${colors.accent}``. Trying to get rid of the notion of ``wtwViolet``.